app/main.dev.js

```diff
@@ -492,3 +510,28 @@ if(!app.isDefaultProtocolClient('tidepoolupload')){
 app.on('window-all-closed', () => {
   app.quit();
 });
+
+app.on('activate', () => {
+  // for mac because, normally it's not common to recreate a window in the app
+  if (mainWindow === null) {
+    i18n.use(i18nextBackend);
+    // Initialize
+    if (!i18n.isInitialized) {
+      setLanguage();
+      i18n.init(i18nextOptions, function (err, t) {
+          createWindow();
+      });
+    } else {
+      createWindow();
+    }
+  }
+});
+
+function setLanguage() {
+  let lng = app.getLocale();
```

Comment:
Can we have this default to English for now, and only use the locale if an environment variable (let's say,  `I18N_ENABLED` is set to `true`, similar to how it's currently set up in Tidepool Web)?

Reply:
Hello!. Of course. Maybe you could tell me where in Tidepool Web is similar. Pls.

What I understand is this:

```
var enabled = process.env.I18N_ENABLED || false
if (!enabled) return 
```

Only configure the language by region if the environment variable is `true`. In `utils / config.i18next` it is defaulted to "en".

Reply:
Yes, that sounds right. The Tidepool Web implementation is a bit different, as it's possible to also select the language from an account form and store that as part of the user preferences, but we can add that functionality to Uploader at a later date.